Allow Codespaces/Gitpod container startup, even if site build fails the `--strict` mode tests

### What would you like to be able to do?

A recent Codespaces container creation failed with:

```
devcontainer process exited with exit code 1
Failed to create container.
Error: Command failed: /bin/sh -c .devcontainer/post-create.sh
Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)
Container creation failed.

Creating recovery container
```

The Codespace I'm trying to start is the one where I'm deleting the `index.md` file, as we're moving that to the main repository. The `mkdocs build` fails, as it _should_, and warns me about the broken links: :upside_down_face: 

```
poetry run mkdocs build
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory:
           /workspaces/renovatebot.github.io/site
INFO    -  Downloading external file:
           https://unpkg.com/mermaid@10.7.0/dist/mermaid.min.js
WARNING -  Doc file 'bot-comparison.md' contains a link
           './index.md#supported-platforms', but the target 'index.md' is not
           found among documentation files.
WARNING -  Doc file 'bot-comparison.md' contains a link
           './index.md#supported-platforms', but the target 'index.md' is not
           found among documentation files.
WARNING -  Doc file 'getting-started/installing-onboarding.md' contains a link
           '../index.md', but the target 'index.md' is not found among
           documentation files.
```

Here's the full log (except the recovery container logs, as those are not relevant here).

<details><summary>Select me to see logs</summary>

```
Host information
----------------
OS: Ubuntu 22.04.3 LTS (stable release)
Image details: https://github.com/github/codespaces-host-images/blob/main/README.md
----------------
Configuration starting...
Cloning...
$ df /var/lib/docker
Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/loop3      32847680 21432056   9721528  69% /var/lib/docker
df process exited with exit code 0
$ git -C "/var/lib/docker/codespacemount/workspace" clone --branch "docs/move-homepage-to-main-repo" --depth 1 https://github.com/HonkingGoose/renovatebot.github.io "/var/lib/docker/codespacemount/workspace/renovatebot.github.io"
Cloning into '/var/lib/docker/codespacemount/workspace/renovatebot.github.io'...
git process exited with exit code 0
$ git -C "/var/lib/docker/codespacemount/workspace/renovatebot.github.io" config --local remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
git process exited with exit code 0
Creating container...
$ devcontainer up --id-label Type=codespaces --workspace-folder /var/lib/docker/codespacemount/workspace/renovatebot.github.io --mount type=bind,source=/.codespaces/agent/mount/cache,target=/vscode --user-data-folder /var/lib/docker/codespacemount/.persistedshare --container-data-folder .vscode-remote/data/Machine --container-system-data-folder /var/vscode-remote --log-level trace --log-format json --update-remote-user-uid-default never --mount-workspace-git-root false --omit-config-remote-env-from-metadata --skip-non-blocking-commands --skip-post-create --config "/var/lib/docker/codespacemount/workspace/renovatebot.github.io/.devcontainer/devcontainer.json" --override-config /root/.codespaces/shared/merged_devcontainer.json --default-user-env-probe loginInteractiveShell --container-session-data-folder /workspaces/.codespaces/.persistedshare/devcontainers-cli/cache --secrets-file /root/.codespaces/shared/user-secrets-envs.json
[281 ms] @devcontainers/cli 0.56.1. Node.js v18.19.0. linux 6.2.0-1019-azure x64.
[1790 ms] Start: Run: docker buildx build --load --build-arg BUILDKIT_INLINE_CACHE=1 -f /tmp/devcontainercli-root/container-features/0.56.1-1714481397330/Dockerfile-with-features -t vsc-renovatebot.github.io-707902b6dea111c21bb791a73af8698ea4e341b6ed9ec4d663bdbe04fd04ce3f --target dev_containers_target_stage --build-arg _DEV_CONTAINERS_BASE_IMAGE=dev_container_auto_added_stage_label /var/lib/docker/codespacemount/workspace/renovatebot.github.io/.devcontainer
#0 building with "default" instance using docker driver

#1 [internal] load .dockerignore
#1 ...

#2 [internal] load build definition from Dockerfile-with-features
#2 transferring dockerfile: 1.46kB done
#2 DONE 0.4s

#1 [internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.5s

#3 [internal] load metadata for ghcr.io/containerbase/devcontainer:10.3.18
#3 ...

#4 [auth] containerbase/devcontainer:pull token for ghcr.io
#4 DONE 0.0s

#3 [internal] load metadata for ghcr.io/containerbase/devcontainer:10.3.18
#3 DONE 1.5s

#5 [dev_container_auto_added_stage_label 1/2] FROM ghcr.io/containerbase/devcontainer:10.3.18@sha256:be9face288702895be509ee8635d90441d93e83572e75acdfaa3bafec452ac78
#5 resolve ghcr.io/containerbase/devcontainer:10.3.18@sha256:be9face288702895be509ee8635d90441d93e83572e75acdfaa3bafec452ac78
#5 resolve ghcr.io/containerbase/devcontainer:10.3.18@sha256:be9face288702895be509ee8635d90441d93e83572e75acdfaa3bafec452ac78 0.2s done
#5 sha256:2f409bbdf2cf401baee65a0424a56185c647c2e9756a9ce2f80db4733cbf66d1 2.67kB / 2.67kB done
#5 sha256:c5255d56d26e8f4fd14fb646a46157aa0650fcc7e9e252d4022f77ce4bba310e 6.91kB / 6.91kB done
#5 sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 0B / 29.53MB 0.0s
#5 sha256:be9face288702895be509ee8635d90441d93e83572e75acdfaa3bafec452ac78 685B / 685B done
#5 sha256:e88ce7c3098b1f8ee89f5491a35cf6f3bbc75694b45c7bcc5a3786fe3916aab4 0B / 374B 0.1s
#5 sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7 0B / 21.24MB 0.2s
#5 sha256:e88ce7c3098b1f8ee89f5491a35cf6f3bbc75694b45c7bcc5a3786fe3916aab4 374B / 374B 0.4s done
#5 sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 9.44MB / 29.53MB 0.6s
#5 sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7 7.34MB / 21.24MB 0.6s
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 0B / 30.83MB 0.6s
#5 sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 18.87MB / 29.53MB 0.9s
#5 sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7 17.83MB / 21.24MB 0.9s
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 4.19MB / 30.83MB 0.9s
#5 sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 24.12MB / 29.53MB 0.9s
#5 sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7 21.24MB / 21.24MB 0.9s
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 8.39MB / 30.83MB 0.9s
#5 sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 29.53MB / 29.53MB 1.1s
#5 sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7 21.24MB / 21.24MB 1.1s done
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 20.97MB / 30.83MB 1.1s
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 25.17MB / 30.83MB 1.2s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 0B / 91.00MB 1.2s
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 30.83MB / 30.83MB 1.4s
#5 sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 29.53MB / 29.53MB 1.4s done
#5 sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 30.83MB / 30.83MB 1.7s done
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 15.22MB / 91.00MB 1.8s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 20.97MB / 91.00MB 1.9s
#5 extracting sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 0.1s
#5 sha256:bea3a71b0a33f58ed1837a649f7d0255f3160e60704e39d7c45e0627ea5f1292 0B / 189B 1.9s
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 0B / 22.34MB 1.9s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 29.36MB / 91.00MB 2.0s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 34.60MB / 91.00MB 2.1s
#5 sha256:bea3a71b0a33f58ed1837a649f7d0255f3160e60704e39d7c45e0627ea5f1292 189B / 189B 2.1s done
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 39.85MB / 91.00MB 2.3s
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 3.15MB / 22.34MB 2.3s
#5 sha256:8eb64bb746ad7be96dc4d2a59a06495b1343140880320b39565dcb6a71586938 0B / 158B 2.3s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 46.14MB / 91.00MB 2.3s
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 7.34MB / 22.34MB 2.3s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 51.38MB / 91.00MB 2.4s
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 12.58MB / 22.34MB 2.4s
#5 sha256:8eb64bb746ad7be96dc4d2a59a06495b1343140880320b39565dcb6a71586938 158B / 158B 2.4s done
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 17.83MB / 22.34MB 2.5s
#5 sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 0B / 18.02MB 2.5s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 60.82MB / 91.00MB 2.7s
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 22.34MB / 22.34MB 2.7s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 73.40MB / 91.00MB 2.9s
#5 sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 22.34MB / 22.34MB 2.8s done
#5 sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 2.98MB / 18.02MB 2.9s
#5 sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 8.39MB / 18.02MB 3.0s
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 0B / 48.03MB 3.0s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 82.84MB / 91.00MB 3.0s
#5 sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 12.58MB / 18.02MB 3.0s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 91.00MB / 91.00MB 3.3s
#5 sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 18.02MB / 18.02MB 3.3s
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 4.19MB / 48.03MB 3.4s
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 7.34MB / 48.03MB 3.5s
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 16.78MB / 48.03MB 3.5s
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 25.50MB / 48.03MB 3.7s
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 31.46MB / 48.03MB 3.8s
#5 sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 91.00MB / 91.00MB 3.8s done
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 36.70MB / 48.03MB 3.9s
#5 sha256:c3214f8d61cb9f38888e9d2b813c46da9e0c9e0ed16bdfd72b6fc3981b97b92f 0B / 283B 3.9s
#5 extracting sha256:e311a697a4031e52691ab1b5a8c325a448280fef9fc03d89dd97ab40f4245bce 2.2s done
#5 sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 18.02MB / 18.02MB 4.1s done
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 46.14MB / 48.03MB 4.1s
#5 sha256:c3214f8d61cb9f38888e9d2b813c46da9e0c9e0ed16bdfd72b6fc3981b97b92f 283B / 283B 4.3s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 0B / 102.32MB 4.3s
#5 extracting sha256:e88ce7c3098b1f8ee89f5491a35cf6f3bbc75694b45c7bcc5a3786fe3916aab4
#5 sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 48.03MB / 48.03MB 4.6s done
#5 sha256:c3214f8d61cb9f38888e9d2b813c46da9e0c9e0ed16bdfd72b6fc3981b97b92f 283B / 283B 4.7s done
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 8.39MB / 102.32MB 4.7s
#5 extracting sha256:e88ce7c3098b1f8ee89f5491a35cf6f3bbc75694b45c7bcc5a3786fe3916aab4 done
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 14.68MB / 102.32MB 4.9s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 20.97MB / 102.32MB 5.0s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 26.21MB / 102.32MB 5.1s
#5 extracting sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 32.51MB / 102.32MB 5.3s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 38.80MB / 102.32MB 5.5s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 47.19MB / 102.32MB 5.6s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 53.48MB / 102.32MB 5.7s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 62.91MB / 102.32MB 5.9s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 77.59MB / 102.32MB 6.3s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 83.89MB / 102.32MB 6.4s
#5 extracting sha256:1ca8028d0988cb4a7848e5099086bcbf70d02c97a9773827fffd1e5e6eebbfb7 1.1s done
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 92.27MB / 102.32MB 6.5s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 102.32MB / 102.32MB 7.0s
#5 sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 102.32MB / 102.32MB 7.2s done
#5 extracting sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a
#5 extracting sha256:96511ffc04d33e6b75c07c8649c848dfedb5b6fead9e61ed79d198f5b780897a 0.9s done
#5 extracting sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179
#5 extracting sha256:4e9f196760834ddafdee86c06f3a6cbe7c45bcedcb813d621076296a3d31d179 2.7s done
#5 extracting sha256:bea3a71b0a33f58ed1837a649f7d0255f3160e60704e39d7c45e0627ea5f1292
#5 extracting sha256:bea3a71b0a33f58ed1837a649f7d0255f3160e60704e39d7c45e0627ea5f1292 done
#5 extracting sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df
#5 extracting sha256:bd84c686bdb8b39eb412359ee600f7abdd7e989609f3229a57b4323dbccd43df 1.0s done
#5 extracting sha256:8eb64bb746ad7be96dc4d2a59a06495b1343140880320b39565dcb6a71586938
#5 extracting sha256:8eb64bb746ad7be96dc4d2a59a06495b1343140880320b39565dcb6a71586938 done
#5 extracting sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1
#5 extracting sha256:57f77f0a9e77f206c987c636802baca1f1a485c74e16d265ee0f9a45865535e1 0.3s done
#5 extracting sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8
#5 extracting sha256:a2abd475e65e76f0b61eab7e9605d07be678f903ad9aa644c75064994db599b8 2.4s done
#5 extracting sha256:c3214f8d61cb9f38888e9d2b813c46da9e0c9e0ed16bdfd72b6fc3981b97b92f
#5 extracting sha256:c3214f8d61cb9f38888e9d2b813c46da9e0c9e0ed16bdfd72b6fc3981b97b92f done
#5 extracting sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce
#5 extracting sha256:9d5df82f32739d5320a76dd29b8ee85ab00fee49274d2face7569918dfa201ce 5.1s done
#5 DONE 31.1s

#6 [dev_container_auto_added_stage_label 2/2] RUN install-tool poetry 1.8.2
#6 0.957 [12:50:32.103] INFO (1): Installing tool poetry@1.8.2...
#6 0.990 installing v2 tool poetry v1.8.2
#6 10.64 linking tool poetry v1.8.2
#6 11.00 Poetry (version 1.8.2)
#6 12.02 [12:50:42.213] INFO (1): Installed tool poetry in 10.1s.
#6 DONE 12.6s

#7 preparing layers for inline cache
#7 DONE 2.2s

#8 exporting to image
#8 exporting layers done
#8 writing image sha256:0f0b8ad2ecc2a5a7bf38a2cd175c27e53c03c2ae47fde9f5938199df4c5b0bf0 0.0s done
#8 naming to docker.io/library/vsc-renovatebot.github.io-707902b6dea111c21bb791a73af8698ea4e341b6ed9ec4d663bdbe04fd04ce3f 0.0s done
#8 DONE 0.1s
[49096 ms] Stop: Run: docker buildx build --load --build-arg BUILDKIT_INLINE_CACHE=1 -f /tmp/devcontainercli-root/container-features/0.56.1-1714481397330/Dockerfile-with-features -t vsc-renovatebot.github.io-707902b6dea111c21bb791a73af8698ea4e341b6ed9ec4d663bdbe04fd04ce3f --target dev_containers_target_stage --build-arg _DEV_CONTAINERS_BASE_IMAGE=dev_container_auto_added_stage_label /var/lib/docker/codespacemount/workspace/renovatebot.github.io/.devcontainer
[50909 ms] Start: Run: docker run --sig-proxy=false -a STDOUT -a STDERR --mount type=bind,src=/var/lib/docker/codespacemount/workspace,dst=/workspaces --mount source=/root/.codespaces/shared,target=/workspaces/.codespaces/shared,type=bind --mount source=/var/lib/docker/codespacemount/.persistedshare,target=/workspaces/.codespaces/.persistedshare,type=bind --mount source=/.codespaces/agent/mount,target=/.codespaces/bin,type=bind --mount source=/mnt/containerTmp,target=/tmp,type=bind --mount type=bind,src=/.codespaces/agent/mount/cache,dst=/vscode -l Type=codespaces -e CODESPACES=******** -e ContainerVersion=13 -e RepositoryName=renovatebot.github.io --label ContainerVersion=13 --hostname codespaces-d07a1b --add-host codespaces-d07a1b:127.0.0.1 --cap-add sys_nice --network host --init --entrypoint /bin/sh vsc-renovatebot.github.io-707902b6dea111c21bb791a73af8698ea4e341b6ed9ec4d663bdbe04fd04ce3f -c echo Container started
Container started
Outcome: success User: vscode WorkspaceFolder: /workspaces/renovatebot.github.io
devcontainer process exited with exit code 0
Running blocking commands...
$ devcontainer up --id-label Type=codespaces --workspace-folder /var/lib/docker/codespacemount/workspace/renovatebot.github.io --mount type=bind,source=/.codespaces/agent/mount/cache,target=/vscode --user-data-folder /var/lib/docker/codespacemount/.persistedshare --container-data-folder .vscode-remote/data/Machine --container-system-data-folder /var/vscode-remote --log-level trace --log-format json --update-remote-user-uid-default never --mount-workspace-git-root false --omit-config-remote-env-from-metadata --skip-non-blocking-commands --expect-existing-container --config "/var/lib/docker/codespacemount/workspace/renovatebot.github.io/.devcontainer/devcontainer.json" --override-config /root/.codespaces/shared/merged_devcontainer.json --default-user-env-probe loginInteractiveShell --container-session-data-folder /workspaces/.codespaces/.persistedshare/devcontainers-cli/cache --secrets-file /root/.codespaces/shared/user-secrets-envs.json
[297 ms] @devcontainers/cli 0.56.1. Node.js v18.19.0. linux 6.2.0-1019-azure x64.
Running the postCreateCommand from devcontainer.json...

.devcontainer/post-create.sh
Fixing permissions of /tmp for GitHub Codespaces...
Submodule 'renovate' (********/renovatebot/renovate.git) registered for path 'deps/renovate'
Cloning into '/workspaces/renovatebot.github.io/deps/renovate'...
Submodule path 'deps/renovate': checked out '09a093d7228231950cf04e4641e577ce0e7d5415'
pnpm install --frozen-lockfile
Scope: all 2 workspace projects
Lockfile is up to date, resolution step is skipped
deps/cache-decoder                       | Progress: resolved 1, reused 0,deps/cache-decoder                       | +181 ++++++++++++++++++
deps/cache-decoder                       | Progress: resolved 1, reused 0,Packages: +305
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
deps/cache-decoder                       | Progress: resolved 1, reused 0,Downloading @parcel/rust@2.12.0: 65.58 MB/65.58 MB, doneDownloading @swc/core-linux-x64-gnu@1.4.13: 16.58 MB/16.58 MB, donedeps/cache-decoder                       | Progress: resolved 181, reused Downloading @swc/core-linux-x64-musl@1.4.13: 20.63 MB/20.63 MB, donedeps/cache-decoder                       | Progress: resolved 181, reused deps/cache-decoder                       | Progress: resolved 181, reused Progress: resolved 305, reused 0, downloaded 270, added 305, done
Downloading typescript@5.4.5: 5.83 MB/5.83 MB, done

devDependencies:
+ @testing-library/cypress 10.0.1
+ cypress 13.8.0
+ husky 9.0.11
+ lint-staged 15.2.2
+ prettier 3.2.5
+ prettier-plugin-packagejson 2.5.0
+ typescript 5.4.5

node_modules/.pnpm/cypress@13.8.0/node_modules/cypress: Running postinstalnode_modules/.pnpm/cypress@13.8.0/node_modules/cypress: Running postinstall script, done in 14.2s
. prepare$ husky
└─ Done in 110ms
Done in 24.2s
poetry install --no-root
Creating virtualenv docs-renovatebot-com-R9lsSN1w-py3.12 in /home/vscode/.cache/pypoetry/virtualenvs
Installing dependencies from lock file

Package operations: 33 installs, 0 updates, 0 removals

  - Installing six (1.16.0)
  - Installing markupsafe (2.1.5)
  - Installing mergedeep (1.3.4)
  - Installing platformdirs (4.2.1)
  - Installing python-dateutil (2.9.0.post0)
  - Installing pyyaml (6.0.1)
  - Installing bracex (2.4)
  - Installing certifi (2024.2.2)
  - Installing charset-normalizer (3.3.2)
  - Installing click (8.1.7)
  - Installing ghp-import (2.1.0)
  - Installing idna (3.7)
  - Installing jinja2 (3.1.3)
  - Installing markdown (3.6)
  - Installing mkdocs-get-deps (0.2.0)
  - Installing packaging (24.0)
  - Installing pathspec (0.12.1)
  - Installing pyyaml-env-tag (0.1)
  - Installing urllib3 (2.2.1)
  - Installing watchdog (4.0.0)
  - Installing babel (2.14.0)
  - Installing colorama (0.4.6)
  - Installing mkdocs (1.6.0)
  - Installing mkdocs-material-extensions (1.3.1)
  - Installing natsort (8.4.0)
  - Installing paginate (0.5.6)
  - Installing pygments (2.17.2)
  - Installing pymdown-extensions (10.8.1)
  - Installing regex (2024.4.28)
  - Installing requests (2.31.0)
  - Installing wcmatch (8.5.1)
  - Installing mkdocs-awesome-pages-plugin (2.9.2)
  - Installing mkdocs-material (9.5.20)
bash bin/get-docs.sh
Lockfile is up to date, resolution step is skipped
Packages: +1340
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Downloading @ls-lint/ls-lint@2.2.3: 5.96 MB/5.96 MB, done
Downloading @swc/core-linux-x64-gnu@1.4.17: 16.64 MB/16.64 MB, done
Downloading @swc/core-linux-x64-musl@1.4.17: 20.60 MB/20.60 MB, done
Downloading openpgp@5.11.1: 5.42 MB/5.42 MB, done
Downloading emojibase-data@15.3.0: 6.11 MB/6.11 MB, done
Progress: resolved 1340, reused 212, downloaded 1128, added 1340, done
node_modules/.pnpm/@swc+core@1.4.17/node_modules/@swc/core: Running postinstall script...
node_modules/.pnpm/core-js-pure@3.37.0/node_modules/core-js-pure: Running node_modules/.pnpm/@swc+core@1.4.17/node_modules/@swc/core: Running postinstall script, done in 263mspure@3.37.0/node_modules/core-js-pure: Running node_modules/.pnpm/better-sqlite3@9.5.0/node_modules/better-sqlite3: Runninode_modules/.pnpm/re2@1.20.10/node_modules/re2: Running install script, done in 2.3s
The following dependencies have build scripts that were ignored: dtrace-provider

dependencies:
+ @aws-sdk/client-codecommit 3.563.0
+ @aws-sdk/client-ec2 3.563.0
+ @aws-sdk/client-ecr 3.563.0
+ @aws-sdk/client-rds 3.563.0
+ @aws-sdk/client-s3 3.563.0
+ @aws-sdk/credential-providers 3.563.0
+ @breejs/later 4.2.0
+ @cdktf/hcl2json 0.20.7
+ @opentelemetry/api 1.8.0
+ @opentelemetry/context-async-hooks 1.23.0
+ @opentelemetry/exporter-trace-otlp-http 0.50.0
+ @opentelemetry/instrumentation 0.50.0
+ @opentelemetry/instrumentation-bunyan 0.37.0
+ @opentelemetry/instrumentation-http 0.50.0
+ @opentelemetry/resources 1.23.0
+ @opentelemetry/sdk-trace-base 1.23.0
+ @opentelemetry/sdk-trace-node 1.23.0
+ @opentelemetry/semantic-conventions 1.23.0
+ @qnighy/marshal 0.1.3
+ @renovatebot/kbpgp 3.0.1
+ @renovatebot/osv-offline 1.5.4
+ @renovatebot/pep440 3.0.20
+ @renovatebot/ruby-semver 3.0.23
+ @sindresorhus/is 4.6.0
+ @types/better-sqlite3 7.6.10
+ @types/ms 0.7.34
+ @types/tmp 0.2.6
+ @yarnpkg/core 4.0.3
+ @yarnpkg/parsers 3.0.0
+ agentkeepalive 4.5.0
+ aggregate-error 3.1.0
+ auth-header 1.0.0
+ aws4 1.12.0
+ azure-devops-node-api 13.0.0
+ better-sqlite3 9.5.0
+ bunyan 1.8.15
+ cacache 18.0.2
+ cacheable-lookup 5.0.4
+ chalk 4.1.2
+ changelog-filename-regex 2.0.1
+ clean-git-ref 2.0.1
+ commander 12.0.0
+ conventional-commits-detector 1.0.3
+ cron-parser 4.9.0
+ deepmerge 4.3.1
+ dequal 2.0.3
+ detect-indent 6.1.0
+ editorconfig 2.0.0
+ email-addresses 5.0.0
+ emoji-regex 10.3.0
+ emojibase 15.3.0
+ emojibase-regex 15.3.0
+ extract-zip 2.0.1
+ find-packages 10.0.4
+ find-up 5.0.0
+ fs-extra 11.2.0
+ git-url-parse 14.0.0
+ github-url-from-git 1.5.0
+ glob 10.3.12
+ global-agent 3.0.0
+ good-enough-parser 1.1.23
+ google-auth-library 9.8.0
+ got 11.8.6
+ graph-data-structure 3.5.0
+ handlebars 4.7.8
+ ignore 5.3.1
+ ini 4.1.2
+ js-yaml 4.1.0
+ json-dup-key-validator 1.0.3
+ json-stringify-pretty-compact 3.0.0
+ json5 2.2.3
+ jsonata 2.0.4
+ klona 2.0.6
+ lru-cache 10.2.0
+ luxon 3.4.4
+ markdown-it 13.0.2
+ markdown-table 2.0.0
+ minimatch 9.0.4
+ moo 0.5.2
+ ms 2.1.3
+ nanoid 3.3.7
+ node-html-parser 6.1.13
+ p-all 3.0.0
+ p-map 4.0.0
+ p-queue 6.6.2
+ p-throttle 4.1.1
+ parse-link-header 2.0.0
+ prettier 3.2.5
+ redis 4.6.13
+ remark 13.0.0
+ remark-github 10.1.0
+ safe-stable-stringify 2.4.3
+ semver 7.6.0
+ semver-stable 3.0.0
+ semver-utils 1.1.4
+ shlex 2.1.2
+ simple-git 3.24.0
+ slugify 1.6.6
+ source-map-support 0.5.21
+ toml-eslint-parser 0.9.3
+ traverse 0.6.9
+ tslib 2.6.2
+ upath 2.0.1
+ url-join 4.0.1
+ validate-npm-package-name 5.0.0
+ vuln-vects 1.1.0
+ xmldoc 1.3.0
+ zod 3.23.4

optionalDependencies:
+ openpgp 5.11.1
+ re2 1.20.10

devDependencies:
+ @hyrious/marshal 0.3.3
+ @jest/environment 29.7.0
+ @jest/globals 29.7.0
+ @jest/reporters 29.7.0
+ @jest/test-result 29.7.0
+ @jest/types 29.6.3
+ @ls-lint/ls-lint 2.2.3
+ @openpgp/web-stream-tools 0.0.14
+ @renovate/eslint-plugin <- @renovatebot/eslint-plugin
+ @semantic-release/exec 6.0.3
+ @swc/core 1.4.17
+ @types/auth-header 1.0.6
+ @types/aws4 1.11.6
+ @types/breejs__later 4.1.5
+ @types/bunyan 1.8.11
+ @types/cacache 17.0.2
+ @types/callsite 1.0.34
+ @types/changelog-filename-regex 2.0.2
+ @types/clean-git-ref 2.0.2
+ @types/common-tags 1.8.4
+ @types/conventional-commits-detector 1.0.2
+ @types/diff 5.2.0
+ @types/eslint 8.56.10
+ @types/fs-extra 11.0.4
+ @types/git-url-parse 9.0.3
+ @types/github-url-from-git 1.5.3
+ @types/global-agent 2.1.3
+ @types/ini 4.1.0
+ @types/js-yaml 4.0.9
+ @types/json-dup-key-validator 1.0.2
+ @types/linkify-markdown 1.0.3
+ @types/lodash 4.17.0
+ @types/luxon 3.4.2
+ @types/markdown-it 13.0.7
+ @types/markdown-table 2.0.0
+ @types/marshal 0.5.3
+ @types/mdast 3.0.15
+ @types/moo 0.5.9
+ @types/node 18.19.31
+ @types/parse-link-header 2.0.3
+ @types/semver 7.5.8
+ @types/semver-stable 3.0.2
+ @types/semver-utils 1.1.3
+ @types/tar 6.1.13
+ @types/traverse 0.6.36
+ @types/unist 2.0.10
+ @types/url-join 4.0.3
+ @types/validate-npm-package-name 4.0.2
+ @types/xmldoc 1.1.9
+ @typescript-eslint/eslint-plugin 7.7.1
+ @typescript-eslint/parser 7.7.1
+ aws-sdk-client-mock 4.0.0
+ callsite 1.0.0
+ common-tags 1.8.2
+ conventional-changelog-conventionalcommits 7.0.2
+ diff 5.2.0
+ emojibase-data 15.3.0
+ eslint 8.57.0
+ eslint-formatter-gha 1.4.3
+ eslint-import-resolver-typescript 3.6.1
+ eslint-plugin-import 2.29.1
+ eslint-plugin-jest 27.9.0
+ eslint-plugin-jest-formatting 3.1.0
+ eslint-plugin-promise 6.1.1
+ eslint-plugin-typescript-enum 2.1.0
+ expect 29.7.0
+ expect-more-jest 5.5.0
+ graphql 16.8.1
+ husky 9.0.11
+ jest 29.7.0
+ jest-extended 4.0.2
+ jest-mock 29.7.0
+ jest-mock-extended 3.0.6
+ jest-snapshot 29.7.0
+ markdownlint-cli2 0.13.0
+ memfs 4.8.2
+ nock 13.5.4
+ npm-run-all2 6.1.2
+ nyc 15.1.0
+ pretty-format 29.7.0
+ rimraf 5.0.5
+ semantic-release 22.0.12
+ tar 6.2.1
+ tmp-promise 3.0.3
+ ts-jest 29.1.2
+ ts-node 10.9.2
+ type-fest 4.18.0
+ typescript 5.4.5
+ unified 9.2.2

. prepare$ run-s 'prepare:*'
│ > renovate@0.0.0-semantic-release prepare:husky /workspaces/renovatebot.git…
│ > husky
[2 lines collapsed]│4> renovate@0.0.0-semantic-release prepare:generate /workspaces/renovateb│7> renovate@0.0.0-semantic-release generate:imports /workspaces/renovateb│ > data/debian-distro-info.json│ > data/kubernetes-api.json5│ > data/node-js-schedule.json
│ > data/ubuntu-distro-info.json
│ > node_modules/emojibase-data/en/shortcodes/github.json
│ generating hashes
│ > renovate@0.0.0-semantic-release prepare:re2 /workspaces/renovatebot.githu…renovate@0.0.0-semantic-release prepare:re2 /workspaces/renovatebot.gi│ > node tools/check-re2.mjs│ Checking re2 ... └─ Done in 2.6s
Done in 21.9s

> renovate@0.0.0-semantic-release build:docs /workspaces/renovatebot.github.io/deps/renovate
> ts-node tools/generate-docs.ts

 INFO: generating docs
 INFO: * static
 INFO: * fetching open GitHub issues
 INFO: * platforms
 INFO: * versionings
 INFO: * datasources
 INFO: * managers
 INFO: * managers/asdf/supported-plugins
 INFO: * presets
 INFO: * templates
 INFO: * configuration-options
 INFO: * self-hosted-configuration
 INFO: * json-schema
bash bin/build-decoder.sh
Scope: all 2 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +181
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 181, reused 181, downloaded 0, added 0, done
node_modules/.pnpm/msgpackr-extract@3.0.2/node_modules/msgpackr-extract: Rnode_modules/.pnpm/msgpackr-extract@3.0.2/node_modules/msgpackr-extract: Running install script, done in 163ms
node_modules/.pnpm/@swc+core@1.4.13/node_modules/@swc/core: Running postinnode_modules/.pnpm/@swc+core@1.4.13/node_modules/@swc/core: Running postinstall script, done in 494ms
../.. prepare$ husky
└─ Done in 83ms
node_modules/.pnpm/lmdb@2.8.5/node_modules/lmdb: Running install script, done in 124ms
Done in 2.9s

> repo-cache-decoder@ build /workspaces/renovatebot.github.io/deps/cache-decoder
> parcel build --dist-dir=../../docs/cache-decoder --public-url /cache-decoder

✨ Built in 2.65s

../../docs/cache-decoder/index.html              1023 B    1.25s
../../docs/cache-decoder/index.fabe2729.js    116.58 KB    1.18s
../../docs/cache-decoder/index.d67eca4e.js    116.57 KB    1.18s
poetry run mkdocs build
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory:
           /workspaces/renovatebot.github.io/site
INFO    -  Downloading external file:
           https://unpkg.com/mermaid@10.7.0/dist/mermaid.min.js
WARNING -  Doc file 'bot-comparison.md' contains a link
           './index.md#supported-platforms', but the target 'index.md' is not
           found among documentation files.
WARNING -  Doc file 'bot-comparison.md' contains a link
           './index.md#supported-platforms', but the target 'index.md' is not
           found among documentation files.
WARNING -  Doc file 'getting-started/installing-onboarding.md' contains a link
           '../index.md', but the target 'index.md' is not found among
           documentation files.
INFO    -  Downloading external file:
           https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.0.3/assets/svg/1f604.svg
INFO    -  Downloading external file:
           https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc3CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc-CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc2CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc5CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc1CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc0CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TjASc6CsQ.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xFIzIFKw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xMIzIFKw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xEIzIFKw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xLIzIFKw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xHIzIFKw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xGIzIFKw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOkCnqEu92Fr1Mu51xIIzI.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic3CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic-CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic2CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic5CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic1CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic0CsTKlA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOjCnqEu92Fr1Mu51TzBic6CsQ.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fCRc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fABc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fCBc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fBxc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fCxc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fChc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmSU5fBBc4.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu72xKOzY.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu5mxKOzY.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu7mxKOzY.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu4WxKOzY.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu7WxKOzY.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu7GxKOzY.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOmCnqEu92Fr1Mu4mxK.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfCRc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfABc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfCBc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfBxc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfCxc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfChc4EsA.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/roboto/v30/KFOlCnqEu92Fr1MmWUlfBBc4.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xdDF4xlVMF-BfR8bXMIjhOsXG-q2oeuFoqFrlnAIe2Imhk1T8rbociImtEluUlYIw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xdDF4xlVMF-BfR8bXMIjhOsXG-q2oeuFoqFrlnAIe2Imhk1T8rbociImtEn-UlYIw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xdDF4xlVMF-BfR8bXMIjhOsXG-q2oeuFoqFrlnAIe2Imhk1T8rbociImtEmOUlYIw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xdDF4xlVMF-BfR8bXMIjhOsXG-q2oeuFoqFrlnAIe2Imhk1T8rbociImtElOUlYIw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xdDF4xlVMF-BfR8bXMIjhOsXG-q2oeuFoqFrlnAIe2Imhk1T8rbociImtEleUlYIw.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xdDF4xlVMF-BfR8bXMIjhOsXG-q2oeuFoqFrlnAIe2Imhk1T8rbociImtEm-Ul.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xTDF4xlVMF-BfR8bXMIhJHg45mwgGEFl0_3vrtSM1J-gEPT5Ese6hmHSV0mf0h.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xTDF4xlVMF-BfR8bXMIhJHg45mwgGEFl0_3vrtSM1J-gEPT5Ese6hmHSx0mf0h.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xTDF4xlVMF-BfR8bXMIhJHg45mwgGEFl0_3vrtSM1J-gEPT5Ese6hmHSt0mf0h.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xTDF4xlVMF-BfR8bXMIhJHg45mwgGEFl0_3vrtSM1J-gEPT5Ese6hmHSd0mf0h.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xTDF4xlVMF-BfR8bXMIhJHg45mwgGEFl0_3vrtSM1J-gEPT5Ese6hmHSZ0mf0h.woff2
INFO    -  Downloading external file:
           https://fonts.gstatic.com/s/robotomono/v23/L0xTDF4xlVMF-BfR8bXMIhJHg45mwgGEFl0_3vrtSM1J-gEPT5Ese6hmHSh0mQ.woff2

Aborted with 3 warnings in strict mode!
make: *** [Makefile:19: build-docs] Error 1
[102658 ms] postCreateCommand failed with exit code 2. Skipping any further user-provided commands.

{"outcome":"error","message":"Command failed: /bin/sh -c .devcontainer/post-create.sh","description":"The postCreateCommand in the devcontainer.json failed.","containerId":"837f288d0b43dee75c7f585661bf4fffe0e704c24da41ef7be652d22ea3666f2"}
Error: Command failed: /bin/sh -c .devcontainer/post-create.sh
    at wY (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:235:130)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async nl (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:4393)
    at async tl (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:3738)
    at async rl (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:2942)
    at async fs (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:2386)
    at async q$ (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:464:1488)
    at async iK (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:464:960)
    at async gAA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:481:3660)
    at async BC (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:481:4775)
devcontainer process exited with exit code 1
Failed to create container.
Error: Command failed: /bin/sh -c .devcontainer/post-create.sh
Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)
Container creation failed.

Creating recovery container

...
```

</details>

I think the main thing we should change is make it so that the devcontainer/codespace is always alllowed to start, even if there are problems with the docs site generation.

### Did you already have any implementation ideas?

I think we need to make it so the `--strict` mode of `mkdocs` does not apply to the Codespaces/Gitpod/Devcontainer startup sequence.

We should _keep_ the strict mode _on_ for the official builds, and for the checks we run on any incoming PRs!

### Do you want to write this feature yourself?

No

--- a/.devcontainer/post-create.sh
+++ b/.devcontainer/post-create.sh
@@ -8,4 +8,4 @@ if [[ "${CODESPACES}" == true ]]; then
 fi
 
 git submodule update --init
-COREPACK_ENABLE_DOWNLOAD_PROMPT=0 make
+COREPACK_ENABLE_DOWNLOAD_PROMPT=0 make STRICT=false

--- a/.gitpod.yml
+++ b/.gitpod.yml
@@ -3,7 +3,7 @@ image:
   context: .gitpod
 tasks:
   - init: >
-      COREPACK_ENABLE_DOWNLOAD_PROMPT=0 make
+      COREPACK_ENABLE_DOWNLOAD_PROMPT=0 make STRICT=false
     command: make serve
 
 ports:

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,11 @@
 default: install build
 
+STRICT?=true
+
+ifeq ($(STRICT),true)
+  build_args=--strict
+endif
+
 install:
 	pnpm install --frozen-lockfile
 	poetry install --no-root
@@ -16,7 +22,7 @@ get-docs:
 	bash bin/get-docs.sh
 
 build-docs:
-	poetry run mkdocs build
+	poetry run mkdocs build $(build_args)
 
 prepare: get-docs
 

--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -2,7 +2,7 @@
 
 # Halt processing when a warning is raised when building the docs.
 # https://www.mkdocs.org/user-guide/configuration/#strict
-strict: true
+strict: false
 
 # Use MkDocs maintainer's recommended strictness settings for link validation.
 # https://www.mkdocs.org/user-guide/configuration/#validation

